api/src/main/java/org/apache/iceberg/encryption/KmsClient.java

```diff
@@ -22,7 +22,8 @@
 import java.nio.ByteBuffer;
 import java.util.Map;
 
-/** A minimum client interface to connect to a key management service (KMS). */
+/** @deprecated A minimum client interface to connect to a key management service (KMS). */
```

Comment:
Add a note that use KeyManagementClient instead ?

Reply:
This should also state when the API will be removed (in 2.0.0).